build/pack.cake

```diff
@@ -32,6 +39,7 @@ Task("Test")
     .Does<BuildParameters>((parameters) =>
 {
     var frameworks = new[] { parameters.CoreFxVersion, parameters.FullFxVersion };
+    var testResultsPath = parameters.Paths.Directories.TestResultsOutput + "/";
```

Comment:
I prefer using `Path.Combine` to build paths and also think the path can be built entirely up here instead of when it's used?

```suggestion
    var testResultsPath = Path.Combine(parameters.Paths.Directories.TestResultsOutput, "vsix.results.xml");
    var absoluteTestResultsPath = MakeAbsolute(new FilePath(testResultsPath));
```

Reply:
Good suggestion, I'll review the code and update where possible to Path.Combine or use the Cake extensions for that.

Reply:
Actually `testResultsPath` is used below when computing the path for nunit test results